sdk/python/kfp/compiler/compiler.py

```diff
@@ -24,8 +24,11 @@
 from typing import Callable, Set, List, Text, Dict, Tuple, Any, Union, Optional
 
 import kfp
+from kfp import dsl
```

Comment:
There are seem to be ~2000 changes in this file.
Is there a way to make this easier to review?
Perhaps you can first create a style PR where you format the files you're going to change in this PR.
WDYT?

Reply:
Done.